### PR TITLE
fix: cli test wrong working directory

### PR DESCRIPTION
**Pull request by bzp2010**

### CODE REVIEW

**Review by tokers — APPROVED** [review]

**Comment by bzp2010** [comment]
@tokers It looks like the CI timed out for some strange reason, can you help me re-run it?

**Comment by tokers** [comment]
@bzp2010 The log message are not enough to troubleshoot the cause. I just re-run all actions.

**Comment by tokers** [comment]
@bzp2010 I'm afraid the test cases are wrong. You may have to troubleshoot it, it's still timed out after re-run.

**Comment by bzp2010** [comment]
Hi, @tokers. I have changed to use systemd as the service manager, and now It don't have blocking the shell and causing the CI to execute indefinitely.

**Comment by bzp2010** [comment]
ping @tokers

File: test/shell/cli_test.sh [hunk 1]
```diff
@@ -18,13 +18,13 @@
 #
 
 start_adapter() {
-  run ./etcd-adapter
+  run systemctl start etcd-adapter
```
Comment (tokers) [inline test/shell/cli_test.sh:21]: Where is the command definition of `run`?

Comment (bzp2010) [reply test/shell/cli_test.sh:21]: @tokers Which is defined by the https://github.com/sstephenson/bats.git framework. 

https://github.com/sstephenson/bats/blob/03608115df2071fff4eaaff1605768c275e5f81f/libexec/bats-exec-test#L50

**Review by tokers — APPROVED** [review]

**Comment by bzp2010** [comment]
Hi, @starsz. Can you help to check again, if there is no problem, please help to merge it, other PRs are relying on this.

**Pull request merged**